src/axolotl/train.py

```diff
@@ -181,7 +181,10 @@ def terminate_handler(_, __, model):
         model.save_pretrained(cfg.output_dir, safe_serialization=safe_serialization)
 
     if not cfg.hub_model_id:
-        trainer.create_model_card(model_name=cfg.output_dir.lstrip("./"))
+        trainer.create_model_card(
+            model_name=cfg.output_dir.lstrip("./"),
+            dataset_tags=list(map(lambda d: d["path"], cfg["datasets"])),
```

Comment:
I like this, I do think we should try to figure out if a path is a local path or hf dataset though.

Reply:
checking if local dir now, should be good. I tried to also check if we were saving the tokenized dataset to hfhub too, but when I enabled `push_dataset_to_hub` I would get 
```
huggingface_hub.utils._validators.HFValidationError: Repo id must be in the form 'repo_name' or 'namespace/repo_name': 'ittailup/alpaca-test/942a2dcbc346823803788fd18b4bb1bf'. Use `repo_type` argument if needed.
```

will report as separate issue.